examples/lighting-app/telink/src/AppTask.cpp

```diff
@@ -58,7 +59,9 @@ CHIP_ERROR AppTask::Init(void)
 {
     SetExampleButtonCallbacks(LightingActionEventHandler);
     InitCommonParts();
-
+#if (!CONFIG_PWM)
```

Comment:
It shouldn't be here. Just override Link LED method.

Reply:
done. I've added it to AppTask class